firstvoices/backend/tests/utils.py

```diff
@@ -159,3 +161,40 @@ def get_batch_import_test_dataset(filename):
     file = open(path, "rb").read().decode("utf-8-sig")
     data = tablib.Dataset().load(file, format="csv")
     return data
+
+
+class TransactionOnCommitMixin:
+    @classmethod
```

Comment:
This is used by several different types of tests, so i moved it up into the test utils rather than having tests reach into other sibling packages